packages/rspack-plugin-minify/package.json

```diff
@@ -25,9 +25,9 @@
     "access": "public"
   },
   "dependencies": {
-    "esbuild": "0.16.3",
```

Comment:
please don't upgrade esbuild and terser version in this PR, since it may introduce breaking change